src/DocumentFormat.OpenXml/GeneratedCode/schemas_openxmlformats_org_wordprocessingml_2006_main.g.cs

```diff
@@ -66469,23 +66469,23 @@ public enum CompatSettingNameValues
         ///<summary>
         ///useWord2013TrackBottomHyphenation.
         ///<para>When the item is serialized out as xml, its value is "useWord2013TrackBottomHyphenation".</para>
-        ///<para>This item is only available in Office2016.</para>
+        ///<para>This item is only available in Office 2016 and later.</para>
```

Comment:
File name indicates schema from year 2006. Is it correct that this parameter is available only starting in Office 2016? (two more cases in this file)

Reply:
This one is a little different. So the ST_CompatSettingName is an enumeration that is defined in most versions of Word format but this enum value was added later in the same xsd file as all previous versions. So it's correct that it is for a later version of Office.